docs/reference/snapshot-restore/index.asciidoc

```diff
@@ -0,0 +1,87 @@
+[[snapshot-restore]]
+= Snapshot and restore
+
+[partintro]
+--
+
+// tag::snapshot-intro[]
+A snapshot is a backup taken from a running Elasticsearch cluster. You can take
```

Comment:
```suggestion
A _snapshot_ is a backup taken from a running Elasticsearch cluster. You can take
```

Is it worth mentioning that it's a backup taken at a specific point in time?

Reply:
Applied.